src/Avalonia.Themes.Fluent/Accents/Base.xaml

```diff
@@ -6,12 +6,12 @@
     <!-- SystemColor Color Resources (Reflect OS High Contrast Settings) -->
     <Color x:Key="SystemColorButtonFaceColor">#FFF0F0F0</Color>
     <Color x:Key="SystemColorButtonTextColor">#FF000000</Color>
-    <Color x:Key="SystemColorGrayText">#FF6D6D6D</Color>
-    <Color x:Key="SystemColorHighlight">#FF3399FF</Color>
-    <Color x:Key="SystemColorHighlightText">#FFFFFFFF</Color>
-    <Color x:Key="SystemColorHotlight">#FF0066CC</Color>
-    <Color x:Key="SystemColorWindow">#FFFFFFFF</Color>
-    <Color x:Key="SystemColorWindowText">#FF000000</Color>
+    <Color x:Key="SystemColorGrayTextColor">#FF6D6D6D</Color>
+    <Color x:Key="SystemColorHighlightColor">#FF3399FF</Color>
+    <Color x:Key="SystemColorHighlightTextColor">#FFFFFFFF</Color>
+    <Color x:Key="SystemColorHotlightColor">#FF0066CC</Color>
+    <Color x:Key="SystemColorWindowColor">#FFFFFFFF</Color>
+    <Color x:Key="SystemColorWindowTextColor">#FF000000</Color>
```

Comment:
According to
https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-theme-resources#windows-system-high-contrast-colors

SystemColorHighlightTextColor is actually used for RadioButton and was missed. Other ones aren't used at all.